### ISSUE TITLE
Couldn't get to use your plugin in rollup+vue+gulp setup

### ISSUE THREAD

**Issue by paulopmx** [vimeo/rollup-plugin-bundle-size#1]
It works in a normal rollup+vue setup, but once I converted everything to gulp which rollup supports your plugin breaks and I get this error

```
TypeError: Path must be a string. Received undefined
    at assertPath (path.js:7:11)
    at Object.basename (path.js:1355:5)
    at Object.ongenerate (/Users/paulo/Sites/swaychat-widget/node_modules/rollup-plugin-bundle-size/index.js:9:32)
```

**Comment by dwightjack** [comment]
I think the problem is here https://github.com/vimeo/rollup-plugin-bundle-size/blob/master/index.js#L9

If I'm correct the `dest` option in rollup has been renamed to `output.file`

**Comment by optunetobi** [comment]
@dwightjack I've added a PR #4 which should fix the issue on newer rollup versions.

**Comment by luwes** [comment]
Fixed in #3 
Today version `v1.0.2` has been published to NPM as well. Sorry for the delay!
